ckanext/canada/triggers.py

```diff
@@ -280,19 +282,21 @@ def update_triggers():
                 IF NEW.record_created IS NULL THEN
                     NEW.record_created := OLD.record_created;
                 END IF;
-                IF NEW.record_modified IS NULL THEN
+                IF NOT sysadmin AND NEW.record_modified IS NULL THEN
                     NEW.record_modified := OLD.record_modified;
                 END IF;
-                IF (req_user_modified = '') IS NOT FALSE THEN
+                IF NOT sysadmin AND (req_user_modified = '') IS NOT FALSE THEN
                     NEW.user_modified := OLD.user_modified;
                 END IF;
                 IF OLD = NEW THEN
                     RETURN NULL;
                 END IF;
-                NEW.record_modified := now() at time zone 'utc';
-                IF (req_user_modified = '') IS NOT FALSE THEN
+                IF NOT sysadmin THEN
```

Comment:
I think we need to record that a sysadmin passed a value for `record_modified` the way we did for `req_user_modified` above so that we can keep that value here instead of overwriting it. That way when sysadmins don't pass a `record_modified` the value will be updated here as normal.